documentation/index.html.md

```diff
@@ -0,0 +1,130 @@
+				$(function() { Flow.initialize(); });
+				</script>
+			</head>
+			<body data-f-model="model.vmf">
+
+			</body>
+		</html>
+
+
+**Notes:**
+
+The `Flow.initialize()` call optionally takes an argument, `channel`, which includes a `run`.
+
+The `run` is an object that includes:
+	
+* `account`: Optional, **User ID** or **Team ID**
+* `project`: Optional, **Project ID**
+* `model`: Optional, name of primary model file (repeated from `data-f-model` in your HTML `<body>` tag)
+
+These parameters are optional because the `run` information defaults correctly based on the location of your user interface file within your Epicenter project and the `data-f-model` element in your `<body>` tag. But you can add it if desired: 
+
+		Flow.initialize({
+			channel: {
+				strategy: 'new-if-initialized',
+				run: {
+					model: 'supply-chain-game.py',
+					account: 'acme-simulations',
+					project: 'supply-chain-game'
+				}
+			}
+		});
+	
+See more details on [customizing the Flow.initialize() method](./generated/flow-js/).
+
+
+<a name="templates"></a>
+####Working with Templates
+
+Several common JavaScript libraries embed a simple template engine, including both `lodash` and `underscore`. Flow.js [already requires](#using_in_project) the `lodash.js` library as one of its dependencies. You can replace `lodash.js` with `underscore.js` if you like. 
+
+Then, start using templates in your project's user interface.
+
+Some basic examples are [adding variables to an enclosing tag](./generated/dom/attributes/binds/default-bind-attr/) and [working with array variables](./generated/dom/attributes/foreach/default-foreach-attr/):
+
+	<div data-f-bind="CurrentYear, Revenue, Profit">
+		In <%= CurrentYear %>, 
+		our company earned <%= Revenue %>, 
+		resulting in <%= Profit %>.
+	</div>
+
+	<ul data-f-foreach="Time">
+		<li> Year <%= index %>: <%= value %> </li>
+	</ul>
+
+But don't feel limited by these examples &mdash; you can use templates throughout your project's user interface code if you're using Flow.js.
+
+Here are a few additional examples to get you started:
+
+	<!-- use templates to perform calculations easily -->
+	
+	You have <strong data-f-bind="Time"><%= 2020 - value %></strong> years remaining in your simulation.
+
+
+	<!-- use templates to display particular content
+			this is particularly useful for multiplayer games
+			safe for Chrome, Firefox, Safari, and IE11+ -->
+	
+	<div id="importantInfo"></div>
+	
+	<script>
+		if (condition) {
+			$(#importantInfo).append("<h2 data-f-bind='Revenue'></h2>");
+		} else {
+			$(#importantInfo).append("<h2 data-f-bind='Sales'></h2>");
+		}
+	</script>
+
+For more background on templates in `lodash.js`, see the <a href="https://lodash.com/docs#template" target="_blank">lodash documentation</a>. If you prefer to use <a href="http://underscorejs.org/#template" target="_blank">underscore</a> instead, that works too &mdash; just be sure to replace `lodash.js` with the `underscore.js` library in your page.
```

Comment:
Minor but I think we mention lodash vs, underscore a bit too often. I'd keep it here and remove the reference up top; most of the people won't care about the difference and the ones that do will know they can replace :)